qiskit/backends/ibmq/ibmqbackend.py

```diff
@@ -61,6 +62,17 @@ def __init__(self, configuration, api=None):
             self._configuration['local'] = False
 
     def run(self, q_job):
+        """Run q_job asynchronously.
+
+        Args:
+            q_job: QuantumJob object
+
+        Returns:
+            IBMQJob (BaseJob)
+        """
+        return IBMQJob(self.run_job, q_job, self._api)
+
```

Comment:
Can we also add a method `get_jobs()` to return all jobs for this backend (for the authenticated user)?